lab_7_llm/start.py

```diff
@@ -15,4 +15,4 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
Why are these changes from lab 7 still here?

Reply:
I reverted all of them